canary/webrtc-c/src/Peer.cpp

```diff
@@ -602,6 +603,13 @@ STATUS Peer::writeFrame(PFrame pFrame, MEDIA_STREAM_TRACK_KIND kind)
             retStatus = STATUS_SUCCESS;
         }
 
+        else if (STATUS_SUCCEEDED(retStatus) && this->firstFrame && this->isMaster) {
```

Comment:
No need to do else if - just do if

Reply:
It will have to be else if because we set `retStatus` to `STATUS_SUCCESS` in the previous if call. I can reorder the ifs, but this should be okay.

Reply:
Oh, of course. I am blind - I thought it simply returns the value there.
 
The suggestion I would have would be something like this:

retStatus = func();

CHK (retStatus == WHAT_TO_CHECK || retStatus == STATUS_SUCCESS, retStatus);

// Reset the retStatus
retStatus = STATUS_SUCCESS;

if (this->firstFrame && blah) {...

Reply:
Ok. This makes it cleaner. Changed.